_posts/2017-09-19-FAF-in-container.markdown

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title:  "FAF in container"
+date:   2017-09-19 10:00:00 +0100
+author: marusak
+visible: 1
+categories: FAF container
+---
+It has never been easier to deploy FAF.
+
+# Why deploying own FAF
+Firstly, standard users probably do not want to deploy own FAF - FAF is here
+to collect a huge amount of crashes from lots of users. Deploying own FAF might
```

Comment:
The word huge is often overused. Consider using a more specific synonym to improve the sharpness of your writing.

an enormous amount